docs/unit.md

```diff
@@ -4,24 +4,71 @@ When running unit tests you should isolate your code so much that it could run i
 
-### aw-test-coverage (node.js)
-`aw-test-coverage` executes **Istanbul** to instrument the code and run tests to create a report of the current test coverage.
+### aw-test-runner (node.js)
+`aw-test-runner` is the default test runner for running unit tests with **Mocha** (using node.js). Appending the argument **cover** will trigger nyc to
+instrument the code before running the tests and make it possible to get test coverage.
 ```
-aw-test-coverage -- test/unit
+aw-test-runner [cover] ./test/unit
 ```
+
 ##### Arguments
-  * `-- "folder"`: The starting point to run code coverage upon.
-  * `-x "folder"`: Excludes this folder from the code coverage.
-  * `--dir "folder"`: Specifies the output folder for the report.
+  * `cover`: This will instrument the code and report the test coverage
+  * `-w`: Watch files for changes which triggers a re-run. (coverage will only be reported after finished test run)
+  * `--opts <file>`: path for mocha options file
+  * `--debug`: The script will 'break' on the very first line. Use the url chrome://inspect to attach to the debug session. It will also show the exact command that will be executed with all arguments
 
-### aw-test-runner (node.js)
-`aw-test-runner` is the default test runner for running unit tests with **Mocha** (using node.js).
+#### Usage recommendations for Nyc together with babel
+If you are using babel as transpiler it's recommended to [set up the project](https://github.com/istanbuljs/nyc#use-with-babel-plugin-istanbul-for-babel-support) with the `babel-plugin-istanbul` and the `cross-env` npm modules.
+
+.babelrc
 ```
-aw-test-runner ./test/unit
+{
+  "babel": {
+    "presets": ["env"],
+    "env": {
+      "test": {
+        "plugins": ["istanbul"]
+      }
+    }
+  }
+}
 ```
 
-##### Arguments
-  * `-w`: Watch files for changes which triggers a re-run.
+package.json
+```
+"scripts": {
+  "test": "cross-env NODE_ENV=test aw-test-runner <tests>"
+}
+```
+
+#### Default Arguments added by after-work.js
+For convenience the runner will append some default arguments:
+
+**Default:**
+  * Mocha:
+    * --compilers js:babel-core/register
+    * --opts dist/config/mocha.opts  (= recursive)
+    * --require dist/config/global.js
+
+**Cover:**
+  * Nyc:
+    * --source-map false (if `babel-plugin-istanbul` is used)
+    * --instrument false (if `babel-plugin-istanbul` is used)
+    * --reporter=text
+    * --reporter=lcov
+    * --all
+    * --include src
+    * --require babel-register
+  * Mocha:
+    * --opts dist/config/mocha.opts (= recursive)
+    * --require dist/config/global.js
+
+#### Override nyc options
+If you want to adjust the arguments for [nyc](https://github.com/istanbuljs/nyc#configuring-nyc) it's recommended to either create a
+.nycrc or add a 'nyc object' in package.json.
+
+#### Override mocha options
+And for adjusting the [mocha](https://mochajs.org/#mochaopts) arguments either create the file /test/mocha.opts or use the --opts <file> argument to aw-test-runner
```

Comment:
Create the file /test/mocha.opts, or use the --opts <file> argument to aw-test-runner for adjusting the [mocha](https://mochajs.org/#mochaopts) arguments.